Doc/library/asyncio-eventloop.rst

```diff
@@ -1,10 +1,15 @@
 .. currentmodule:: asyncio
 
+.. _asyncio-event-loop:
```

Comment:
Please keep the ref where it was.  The page is rather long and the idea was to jump the user right where the event loop methods are defined (skipping the prelude).

Reply:
@1st1 Oh, I see. It seemed a bit inconsistent with the other pages and skipped over the summary, so I thought it was unintentionally misplaced. I'll correct it.